packages/ember-metal/lib/property_get.js

```diff
@@ -13,6 +14,23 @@ const ALLOWABLE_TYPES = {
   string: true
 };
 
+export const PROXY_CONTENT = symbol('PROXY_CONTENT');
+
+export function getPossibleMandatoryProxyValue(obj, keyName) {
+  if (MANDATORY_GETTER && EMBER_METAL_ES5_GETTERS && HAS_NATIVE_PROXY) {
+    let content = obj[PROXY_CONTENT];
+    if (content === undefined) {
+      return obj[keyName];
+    } else {
+      /* global Reflect */
+      return Reflect.get(content, keyName, obj);
```

Comment:
Do we know of platforms that have native `Proxy` but don't support `Reflect`?

Reply:
Doesn't look like it: http://kangax.github.io/compat-table/es6/#test-Reflect (the opposite could be true though)